components/Current-Weather/current-weather.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import style from "../../styles/Home.module.css";
+
+const CurrentWeather = ({data}) => {
+const [currentWeather, setCurrentWeather] = useState("");
```

Comment:
Ah I led you astray earlier! Looks like you don't need this state correct since the current weather data is passed as a prop? Sorry about that!